### PR TITLE
Monkey devour fix

### LINKED ISSUE #25921: Plasmamen are dying while being cloned
I deliberately changed this in my cloning PR, the cloning pod is supposed to be an "air neutral" environment, not interacting with the outside air (because they're floating in clonebrine).

But it's still happening, I must have missed something.

### PR DESCRIPTION
**Pull request by Davidj361**
:cl: MrSavage/PoopLicker/Davidj361
tweak: Changed the way you eat monkeys by clicking the monkey over and over, also refactored the code
/:cl:

[why]: You weren't able to devour monkeys as a fart person anymore.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
>fart person

also i think you linked the wrong issue

also i dont think this is actually a bug, i ate a monkey as a fat guy literally last night

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 1]
```diff
@@ -159,11 +159,13 @@
...
-		devour_mob(pulling)
-	else
-		..()
+// BROKEN
+// This supposed to be the code for devouring monkeys but it became broken
+// /mob/living/carbon/human/grabbedby(mob/living/carbon/user, supress_message = 0)
+// 	if(user == src && pulling && !pulling.anchored && grab_state >= GRAB_AGGRESSIVE && (disabilities & FAT) && ismonkey(pulling))
+// 		devour_mob(pulling)
+// 	else
+// 		..()
```
Comment (PKPenguin321) [inline code/modules/mob/living/carbon/human/human_defense.dm]: dont comment out code like this and also this isnt really broken this is a fake fix

**Comment by Davidj361** [comment]
Alright the code is more of a refactor I guess. I thought it was broken because I was told I had to drag the monkey onto me to eat it. Apparently you have to be in aggressive grab **then click yourself**. In my edit, you just keep clicking for grabs and you start devouring. In my opinion the clicking yourself thing seems like a fluke fix. But regardless, I really don't like the old code on how it read. It made no sense as it looked completely backwards with how it read intuitively.

**Comment by PKPenguin321** [comment]
The issue you list this as a fix for is `Plasmamen are dying while being cloned #25921` which is wrong
This does not fix a bug and introduces a bug where you cannot grab things regularly if they are devour-able, and instead are only able to devour them

**Comment by Davidj361** [comment]
Yeah I copy pasted and am a bit tired. Anyhow, you can delete this PR because after closer look, the code works as it is meant to but it is not clear at all to read.

**Pull request closed without merging**